src/entities/Proposal/routes.ts

```diff
@@ -441,6 +463,8 @@ export async function updateProposalStatus(req: WithAuth<Request<{ proposal: str
 
   await ProposalModel.update<ProposalAttributes>(update, { id })
 
+  await commentProposalUpdateInDiscourse(id)
```

Comment:
once you have  this function working with `inBackground` you can remove the `await`

Reply:
Ty, removed it here and from proposal jobs